aiapy/calibrate/util.py

```diff
@@ -248,6 +319,51 @@ def get_error_table(error_table=None):
     return table
 
 
-@manager.require("error_table", *URL_HASH[ERROR_VERSION])
+@manager.require("error_table", *URL_HASH_ERROR_TABLE[ERROR_VERSION])
 def fetch_error_table():
     return manager.get("error_table")
+
+
+@manager.require("pointing_table", *URL_HASH_POINTING_TABLE)
+def fetch_pointing_table():
+    return manager.get("pointing_table")
+
+
+@manager.require("response_table_v10", *URL_HASH_RESPONSE_TABLE[10])
+def fetch_response_table_v10():
+    return manager.get("response_table_v10")
+
+
+@manager.require("response_table_v9", *URL_HASH_RESPONSE_TABLE[9])
+def fetch_response_table_v9():
+    return manager.get("response_table_v9")
+
+
+@manager.require("response_table_v8", *URL_HASH_RESPONSE_TABLE[8])
+def fetch_response_table_v8():
+    return manager.get("response_table_v8")
+
+
+@manager.require("response_table_v7", *URL_HASH_RESPONSE_TABLE[7])
+def fetch_response_table_v7():
+    return manager.get("response_table_v7")
+
+
+@manager.require("response_table_v6", *URL_HASH_RESPONSE_TABLE[6])
+def fetch_response_table_v6():
+    return manager.get("response_table_v6")
+
+
+@manager.require("response_table_v4", *URL_HASH_RESPONSE_TABLE[4])
+def fetch_response_table_v4():
+    return manager.get("response_table_v4")
+
+
+@manager.require("response_table_v3", *URL_HASH_RESPONSE_TABLE[3])
+def fetch_response_table_v3():
+    return manager.get("response_table_v3")
+
+
+@manager.require("response_table_v2", *URL_HASH_RESPONSE_TABLE[2])
+def fetch_response_table_v2():
+    return manager.get("response_table_v2")
```

Comment:
If this is what is necessary to use the data manager for these files, then I think we should not use the data manager. We could always use it later using the delayed aspect that will make it into 6.1

Reply:
Well the issue is that I don't know how best to handle having 8 different versions of the file depending on user input.